### ISSUE TITLE
Simulator starts on the "Offline" mode and does not response to "Go Online"

### ISSUE THREAD

**Issue by manone888** [lgsvl/simulator#1609]
Hi guys, when I launched the simulator, it showed the below screen:
![Screenshot 2021-07-20 09:08:04](https://user-images.githubusercontent.com/19791379/126330705-b379a069-a696-412f-bf14-4d8803a0ca72.png)
It did not have any response when I clicked "Go Online" and "Unlink". But it had response to the "Clear Cache" and "Quit".
I tried the three releases of 2021 and got the same result. But when I tried the release of 2020.06, I could click the "Open browser" button.
Could you please advise on this issue? Thanks a lot.

**Comment by EricBoiseLGSVL** [comment]
what version is the image from 2021.2?  What cloud url are you trying to reach? It should be in the config.yml file in the root.

**Comment by nsrj** [comment]
Thank you for your reply @EricBoiseLGSVL. Yes, it is from 2021.2. Please see the below figure.
![Screenshot](https://user-images.githubusercontent.com/86631353/126724138-75acade0-8e09-4169-b9cd-228d2c70c869.png)
By the way, the OS of the computer is Ubuntu 20.04.

**Comment by manone888** [comment]
@EricBoiseLGSVL, thank you for your reply. Yes, it is from 2021.2. Please see the below figure.
![Screenshot](https://user-images.githubusercontent.com/19791379/126724655-16beb725-56ca-46fc-a974-a0d03bd9858b.png)
By the way, the OS of the computer is Ubuntu 20.04.

**Comment by EricBoiseLGSVL** [comment]
Please navigate to the AppData folder for simulator and delete it and start it again.  Another issue maybe something in the OS is blocking the app from the network.

**Comment by baoxinchen** [comment]
Hi @manone888, are you using a computer without the cable internet interface? Like, gaming laptops. I had the same issue on my razer laptop. If I plug in a USB internet dongle, the simulator could go online on WiFi or wire network. I think this is a BUG on this Linux build.

**Comment by EricBoiseLGSVL** [comment]
Yes, @baoxinchen is correct, direct cable internet is a must.  We have no support for wireless or usb internet hubs.  It would be great if you post a solution for users if they are using this.  Next release we have an update to web socket sharp that may work better.
